manifests/03-rbac-role-cluster.yaml

```diff
@@ -97,3 +97,16 @@ rules:
   - validatingwebhookconfigurations
   verbs:
   - get
+---
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1
+metadata:
+  name: helm-chartrepos-viewer
+rules:
+  - apiGroups:
+      - helm.openshift.io
+    resources:
+      - helmchartrepositories
+    verbs:
+      - get
+      - list
```

Comment:
Looks good to me, all authenticated users should be allowed to 'read' helm chart repositories. That'll naturally drive the content in the developer catalog.

Any advanced configuration needs to be done by the admin.

Reply:
Won't `watch` action be needed?

Reply:
no, because the operator does not watch for changes. The permissions are needed for console `/api/helm/charts/index.yaml` endpoiint - it retrieves the list of all `HelmChartRepository` CRs from the cluster, and use them to retrieve individual `index.yaml` files that get combined at the end.